modules/client-configuration/pages/contact-methods-pushssh.adoc

```diff
@@ -0,0 +1,103 @@
+[[contact-methods-pushssh]]
+= Push via SSH
```

Comment:
The most confusing thing for the customers there that we have actually 2 different methods:
- Push via SSH
- Push via SSH tunnel
These methods are defined with the activation key only as far as I know and only the second one is bringing up the tunnel with `1233` port to the Server. The first one actually expects that there is a direct connection with HTTPS to the Server.

It makes sense to describe it explicitly, because with no creating the activation key the first method is used, but not the method with the tunnel.